apps/backtalk/src/components/backtalk/results/SurveyResults.tsx

```diff
@@ -28,6 +28,7 @@ import {
   useDisclosure,
   Tooltip,
 } from '@chakra-ui/react';
+import { default as NextLink } from 'next/link';
```

Comment:
We actually have a Link component already: https://github.com/ClickPop/aww-rats-client/blob/main/packages/common/components/shared/Link.tsx